### PR TITLE
MSI: Update Openshift Version

### PR DESCRIPTION
**Pull request by chetna14manku**

### CODE REVIEW

**Review by myakove — APPROVED** [review]

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhoai-addon-interop-v4.14-GA-stage-smoke-osd-aws-rhods-addon-baseline-fips

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhoai-addon-interop-v4.14-GA-stage-smoke-osd-aws-rhods-addon-baseline-fips

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhoai-addon-interop__v4.14-GA-stage.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ tests:
       ADDON1_CONFIG: name=managed-odh;notification-email=interop-qe-ms@redhat.com;ocm-env=stage
       ADDONS_OPERATORS_RUN_IN_PARALLEL: "false"
       CLUSTER_NAME_PREFIX: rhods
-      CLUSTER1_CONFIG: platform=rosa;region=us-east-2;version=4.13;channel-group=stable;compute-machine-type=g4dn.2xlarge;timeout=90m;replicas=2;fips=true;ocm-env=stage
+      CLUSTER1_CONFIG: platform=rosa;region=us-east-2;version=4.14;channel-group=stable;compute-machine-type=g4dn.2xlarge;timeout=3h;replicas=2;fips=true;ocm-env=stage
```
Comment (rnetser) [inline ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhoai-addon-interop__v4.14-GA-stage.yaml:40]: why is this change needed?
```;timeout=3h
```

**Review by rnetser — APPROVED** [review]

**Review by rnetser — APPROVED** [review]
